### PR TITLE
Update broken links

### LINKED ISSUE #8597: Dead link
[ListView with Multiple Views Sample](https://archive.msdn.microsoft.com/Project/Download/FileDownload.aspx?ProjectName=wpfsamples&DownloadId=7766) is a dead link

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 277d063e-777f-c1ec-7e86-479f636f1788
* Version Independent ID: d84b065e-5b13-cc11-cfb1-df02969e3d3a
* Content: [ListView Overview](https://docs.microsoft.com/en-us/dotnet/framework/wpf/controls/listview-overview#creating-a-custom-view-mode)
* Content Source: [docs/framework/wpf/controls/listview-overview.md](https://github.com/dotnet/docs/blob/master/docs/framework/wpf/controls/listview-overview.md)
* Product: **dotnet-framework**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by jrstrayhorn**
Updating link to ListView with Multiple Views Sample

## Summary

Updating broken links

Fixes #8597

### CODE REVIEW

**Comment by Thraka** [comment]
@jrstrayhorn Hi! The link you changed to doesn't work for me.

File: docs/framework/wpf/controls/listview-overview.md [hunk 1]
```diff
@@ -69,13 +69,13 @@ The <xref:System.Windows.Controls.ListView> control provides the infrastructure
 ## Sharing the Same View Mode  
  Two <xref:System.Windows.Controls.ListView> controls cannot share the same view mode at the same time. If you try to use the same view mode with more than one <xref:System.Windows.Controls.ListView> control, an exception occurs.  
   
- To specify a view mode that can be simultaneously used by more than one <xref:System.Windows.Controls.ListView>, use templates or styles. For an example of how to define views as <xref:System.Windows.FrameworkElement.Resources%2A>, see [ListView with Multiple Views Sample](https://go.microsoft.com/fwlink/?LinkID=160013).  
+ To specify a view mode that can be simultaneously used by more than one <xref:System.Windows.Controls.ListView>, use templates or styles. For an example of how to define views as <xref:System.Windows.FrameworkElement.Resources%2A>, see [ListView with Multiple Views Sample](https://msdn.microsoft.com/library/ms771469).
```
Comment (rpetrusha) [inline docs/framework/wpf/controls/listview-overview.md]: A better link target for both links is <https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-3.0/ms771469(v=vs.85)>.

**Review by rpetrusha — COMMENTED** [review]
I'm not sure why your link doesn't work, @Thraka; it works for me.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for contributing to the dotnet/docs repo, @jrstrayhorn, and for fixing these broken links. I've suggested a different link target on docs.microsoft.com, though.

File: docs/framework/wpf/controls/listview-overview.md (hunk 1 above)
Comment (jrstrayhorn) [reply docs/framework/wpf/controls/listview-overview.md]: Made the change based on your recommendation.

**Comment by jrstrayhorn** [comment]
@rpetrusha I've made the changes to the link that you requested!!  And glad I could help! 😁

**Comment by jrstrayhorn** [comment]
whooops didn't mean to close.. still learning!

**Comment by mairaw** [comment]
We need to find a replacement in the samples repo for these links:
https://github.com/Microsoft/WPF-Samples 

Sometimes they don't exist 😢

**Comment by Thraka** [comment]
Well the old link is working for me now, but the new link is better 😄 strange!

**Comment by jrstrayhorn** [comment]
@mairaw @rpetrusha hey! I tried to find a better link in https://github.com/Microsoft/WPF-Samples and I didn't see one.. oh well at least I gave it a shot!! 😁  Good experience!

**Comment by rpetrusha** [comment]
Thanks for looking; I couldn't find it either, @jrstrayhorn. The best thing to do is to remove the cross-reference to the sample. Would you like to do that, or would you like us to?

**Comment by jrstrayhorn** [comment]
I can do that, I'll update PR later today!

**Comment by jrstrayhorn** [comment]
@rpetrusha I've removed the broken links!

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for your patience and for removing the links, @jrstrayhorn. I'll merge your PR now. Your changes should be live on docs.microsoft.com in the next day or two.

**Comment by jrstrayhorn** [comment]
@rpetrusha  thank you!! 😎

**Pull request merged**